### ISSUE TITLE
Visual Studio 2015 crashes on NuGet package update

### ISSUE THREAD

**Issue by xperiandri** [NuGet/Home#1897]
Reproduction steps and crash dump attached
[ReproSteps635877920078842170.zip](https://github.com/NuGet/Home/files/81514/ReproSteps635877920078842170.zip)
[ETLTrace635877920078832165.etl](http://rusfolder.com/44615883)

**Comment by yishaigalatzer** [comment]
Thanks for reporting this issue.

@xperiandri what version of NuGet are you using? and what Update of Visual Studio (RTM or Update I). We recommend using NuGet 3.3.

Can you provide repro steps, and callstack? I'm blocked from downloading the zip file you provided.

**Comment by xperiandri** [comment]
Update 1
3.3.0.167
Call stack is not available. Visual Studio instantly closes a window with call stack.
So etl you downloaded, didn't you? Only zip required

**Comment by xperiandri** [comment]
http://rusfolder.com/44617178

**Comment by yishaigalatzer** [comment]
Ok I managed to open the file, this looks like a DNX codepath (that we own a very small piece of). What would really help is repro instructions in plain english and a like to the project you are trying to run (looks like this is a github project for F# DNX).

**Comment by xperiandri** [comment]
That is right.
Clone fsharp-dnx
Open
Manage NuGet packages for solution
Updates
FSharp Compiler
Update to 2.0.0.2

Path to repository is `C:\Users\Андрій\Dev\GitHub\fsharp-dnx`

**Comment by yishaigalatzer** [comment]
@xperiandri the path you specified is on your hard drive? can you share the http url?

**Comment by xperiandri** [comment]
`https://github.com/fsprojects/fsharp-dnx`

**Comment by yishaigalatzer** [comment]
thanks got it and cloning

**Comment by yishaigalatzer** [comment]
Works for me on the latest bits (so 3.4 preview, contact me if you want to give it a try yigalatz @ msft.com)

If this fixes your issue, then we are good. If not lets work on getting a proper repro or callstack from you.

I'm on DNX RC + NuGet 3.4 private build

![dnx-fsharp](https://cloud.githubusercontent.com/assets/1238711/12207187/0b30a046-b5fa-11e5-9464-2a86ef7e7ec2.png)

**Comment by xperiandri** [comment]
Could you share that build?

**Comment by xperiandri** [comment]
OK, looks like it can be closed

**Comment by yishaigalatzer** [comment]
We verified this is fixed in 3.4

**Comment by bunceg** [comment]
This is happening regularly for me and my colleagues - especially when nuget tries to delete a package on disk that is checked into Visual Studio Online. How do I check my NuGet package version?

**Comment by yishaigalatzer** [comment]
Bunceg 
1. This looks unrelated to the bug reported here
2. To see your version use tools->extensions and updates and look at the version
